src/components/Login/LoginForm.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { Formik, ErrorMessage, Form } from "formik";
+import * as Yup from "yup";
+
+import AuthService from "../../services/auth.service";
```

Comment:
jeśli tsconfig jest dobrze skonfigurowany to 
import AuthService from "services/auth.service"

Reply:
ok, poprawiłam i ju nie jest potrzebny